MycroftAI/mycroft-core
# Feature/jony0008 swedish

## Description
Add Swedish dialogues by @jony0008

## How to test
Set language to sv-se or sv-fi and make sure mycroft sounds like the Swedish chef when enabling SSH.

## Contributor license agreement signed?
CLA [Yes]
